frontend/templates/views/partials/analytics.j2

```diff
@@ -20,156 +20,188 @@
   'extraUrlParams': {
     'cd2': '${ampdocHost}'
   },
+  'requests': {
+    "base": "https://google-analytics.com/collect/"
+    "CWV_EVENT": "${base}?v=1&t=event&tid=${gtag_id}&cid=0&ec=cwv
+  },
   'triggers': {
-    'banner': {
-      'selector': '#top-banner',
-      'on': 'click',
+    'default pageview': {
```

Comment:
I'm not sure if we allow whitespaces in trigger names, but also I think we should follow (with what I think is naming convention) and use camel case here.

Reply:
done! thanks